src/bufr.interface.f90

```diff
@@ -1,3 +1,12 @@
+!> @file
```

Comment:
It's not usual to have an extra "." in the filename. Can this file be named bufr_c_interface.f90, to accord with usual conventions?